src/test/java/com/celements/auth/user/listener/AddDefaultGroupsToNewUserListenerTest.java

```diff
@@ -0,0 +1,71 @@
+package com.celements.auth.user.listener;
+
+import static org.easymock.EasyMock.*;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.xwiki.bridge.event.DocumentCreatedEvent;
+import org.xwiki.model.reference.DocumentReference;
+
+import com.celements.auth.user.User;
+import com.celements.auth.user.UserInstantiationException;
+import com.celements.auth.user.UserService;
+import com.celements.common.test.AbstractComponentTest;
+import com.celements.model.access.exception.DocumentSaveException;
+import com.xpn.xwiki.doc.XWikiDocument;
+
+public class AddDefaultGroupsToNewUserListenerTest extends AbstractComponentTest {
+
+  private AddDefaultGroupsToNewUserListener usrListener;
+
+  @Before
+  public void prepareTest() throws Exception {
+    registerComponentMock(UserService.class);
+    usrListener = getBeanFactory().getBean(AddDefaultGroupsToNewUserListener.class);
+  }
+
+  @Test
+  public void testOnEventInternal() throws Exception {
```

Comment:
naming convention:
```suggestion
  public void test_onEventInternal() throws Exception {
```
change all test methods.